how-tos/setup-staging-release.md

```diff
@@ -13,45 +13,22 @@ similar. So mainly rather larger changes to the code base, changes that actually
 
 ## Simulation project branch
 
-Project branch `jamun` has traditionally been our culprit to simulate `beta`. We used `date` for simulating nightlies from `central`.
+Project branch `jamun` has traditionally been our culprit to simulate `beta`. We used `maple` for simulating nightlies from `central`.
 
 Use cases:
 - for `central` -> `beta` staging release we would use `from_repo_url` = `central`  and `to_repo_url` = `jamun`
-- for `date` -> `central` -> `beta` (stuff that lands in `central` and right after that goes in `beta`) staging release we would use `from_repo_url` = `date`  and `to_repo_url` = `jamun`
+- for `maple` -> `central` -> `beta` (stuff that lands in `central` and right after that goes in `beta`) staging release we would use `from_repo_url` = `maple`  and `to_repo_url` = `jamun`
 
 ## Merge scripts
 
-1.
-```sh
-mkdir staging_release_merge
-cd staging_release_merge
-wget https://hg.mozilla.org/build/tools/raw-file/default/buildfarm/utils/archiver_client.py
-python archiver_client.py mozharness --destination mozharness-central --repo mozilla-central --rev default --debug  # Central must be used against every branch
+1. Use the script ../scripts/staging_merge.py to run the merge for you. For example:
```

Comment:
Let's highlight a bit the script here. Either use the `../script` or better add a href to it.